### PR TITLE
Feat/about responsive

### LINKED ISSUE #48: Add error handling (Loading component & Error component)

### LINKED ISSUE #49: Make About page responsive

### PR DESCRIPTION
**Pull request by irmakerdem**
** [Project Ticket](Project ticket link goes here) **
- Resolves #48 
- Resolves #49

** Describe the functionality to be tested: **
- Creates Error and Loading components and adds styling
- Adds desktop responsive design for About page

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: src/Components/Error/Error.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Error.scss';
+
+const Error = () => {
+	return (
+    <div className="error-messaging">
+      <h2>Something went wrong.</h2>
+      <h3>Click the Roe'dToAid logo above to return home.</h3>
+      <img className="error-image" src="https://media.giphy.com/media/oypwltUsyqZT2jNkyU/giphy.gif" alt="This uterus says error!"/>
```
Comment (lkarpins) [inline src/Components/Error/Error.js:9]: Love this view!

File: src/Views/AboutView/Aboutview.scss [hunk 2]
```diff
@@ -1,25 +1,64 @@
...
+  gap: 1.5rem;
 }
 
+.inclusivity-only {
+  flex-direction: column-reverse;
+}
+
+@media(min-width: 768px) {
+  .inclusivity-only {
+    flex-direction: row;
```
Comment (lkarpins) [inline src/Views/AboutView/Aboutview.scss:58]: Yayyy media queries!! 🎉

**Review by lkarpins — APPROVED** [review]
Looks awesome, Irmak! Nice work driving, today!!

File: src/Components/Footer/Footer.js [hunk 3]
```diff
@@ -5,73 +5,71 @@ import logo from "./../../assets/roed-to-aid-logo.svg";
...
+        <img src={logo} alt="logo" className="logo-img" data-cy="logo-img"></img>
+        <div className="footer-team-container">
+          <div className="fe-container">
+            <p data-cy="fe-team">FrontEnd Team:</p>
+            <a
+              href="https://github.com/Rallen13/"
+              target="blank"
+              data-cy="git-hub-link-rachel"
+            >
+              Rachel Allen
```
Comment (catlynbowles) [inline src/Components/Footer/Footer.js:17]: I love the changes you made here.

**Review by catlynbowles — APPROVED** [review]

**Review by Rallen13 — APPROVED** [review]
Looks great Irmak!

**Pull request merged**